frontend/src/employee-frontend/components/unit/tab-calendar/discussion-surveys/times-calendar/CalendarEventTimeInput.tsx

```diff
@@ -0,0 +1,50 @@
+// SPDX-FileCopyrightText: 2017-2024 City of Espoo
+//
+// SPDX-License-Identifier: LGPL-2.1-or-later
+
+import React from 'react'
+import styled from 'styled-components'
+
+import { BoundForm, useFormField, useFormFields } from 'lib-common/form/hooks'
+import { TimeInputF } from 'lib-components/atoms/form/TimeInput'
+import { FixedSpaceRow } from 'lib-components/layout/flex-helpers'
+
+import { calendarEventTimeForm } from '../survey-editor/form'
+
+const InputContainer = styled(FixedSpaceRow)`
+  * > input:focus {
+    box-shadow: none;
+  }
+`
+
+export default React.memo(function CalendarEventTimeInput({
+  bind
+}: {
+  bind: BoundForm<typeof calendarEventTimeForm>
+}) {
+  const timeRange = useFormField(bind, 'timeRange')
+  const { startTime, endTime } = useFormFields(timeRange)
+
+  return (
+    <InputContainer
+      spacing="xxs"
+      alignItems="center"
+      justifyContent="center"
+      data-qa="time-input-container"
+    >
+      <TimeInputF
+        bind={startTime}
+        info={{ text: '' }}
```

Comment:
check if the empty info is necessary or if the rop should be simply omitted

Reply:
Leaving the info prop out will lead to an internal default being used resulting in visible validation texts which is not desired in the cramped day cell components.